applications/Couette.prm

```diff
@@ -103,6 +103,13 @@ subsection Output control parameters
 end
 
 
+subsection Refinement control parameters
```

Comment:
This subsection could use a renaming. Maybe  "Spatial discretization parameters"?